lessons/213/k8s/deploy/go-app/1-deployment.yaml

```diff
@@ -30,13 +30,17 @@ spec:
           valueFrom:
```

Comment:
trade all memory for minimal amount gc cycles?
```
- name: GOGC
   value: off
```

reduce gc cycles knowing ahead we will eat up all memory anyway ?
```
- name: GOGC
   value: 200
```


```
GOGC controls the aggressiveness of the garbage collector. By default this value is assumed to be 100, which means garbage collection will not be triggered until the heap has grown by 100% since the previous collection. Effectively GOGC=100 (the default) means the garbage collector will run each time the live heap doubles.

Setting this value higher, say GOGC=200, will delay the start of a garbage collection cycle until the live heap has grown to 200% of the previous size. Setting the value lower, say GOGC=20 will cause the garbage collector to be triggered more often as less new data can be allocated on the heap before triggering a collection.
```